ci-cd-bootstrap/params/npm-build-pod/build_params

```diff
@@ -0,0 +1,5 @@
+SOURCE_REPOSITORY_URL=https://github.com/InfoSec812/labs-ci-cd
```

Comment:
I presume you  may have a bit of a chicken-and-egg situation here, but could we update the URL to be referencing the `rht-labs` org rather than an individual's repo?

Reply:
I can change it, but then it will be non-functional until the PR is merged.